package.json

```
@@ -47,7 +47,7 @@
     "@fastify/cors": "7.0.0",
     "@fastify/helmet": "8.0.0",
     "@fastify/secure-session": "4.0.0",
-    "@graasp/sdk": "github:graasp/graasp-sdk",
+    "@graasp/sdk": "github:graasp/graasp-sdk#etherpad",
```

- TODO: once https://github.com/graasp/graasp-sdk/pull/69 is merged
```suggestion
    "@graasp/sdk": "github:graasp/graasp-sdk",
```